apps/docs/package.json

```diff
@@ -3,19 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "rspress build && pnpm run build-ladle",
-    "build-ladle": "pnpm --filter=@accelint/design-system exec ladle build -o ../../apps/docs/dist/ladle --base ladle",
+    "dev": "pnpm run build:deps && rspress dev",
+    "build:deps": "pnpm run build:ladle && pnpm run build:typedoc",
+    "build:ladle": "pnpm --filter=@accelint/design-system exec ladle build -o ../../apps/docs/src/public/ladle --base ladle",
+    "build:typedoc": "typedoc",
+    "build": "pnpm run build:deps && rspress build",
+    "preview": "rspress preview",
```

Comment:
Not a requirement, just a preference of mine that can be ignored, I like to order things in alpha-order. There is a command in VSCode to do this for you if, like me, you don't want to do tedious things like this.

Reply:
[updated](https://github.com/gohypergiant/standard-toolkit/pull/146/commits/9bee35cf58d74a45cb0240a9d87bf8ea7f39162c)